### ISSUE TITLE
osx builds?

### ISSUE THREAD

**Issue by jchodera** [conda-forge/ambertools-feedstock#8]
Thanks for maintaining a conda-forge build of AmberTools!

Has there been an effort to build `osx` builds of this feedstock? If not, I wonder if @hainm has some thoughts on how we might be able to get that to work.

**Comment by hainm** [comment]
There is no big different from linux build, except that gfortran-related libraries are included in the distribution.

https://github.com/Amber-MD/ambertools-binary-build/blob/257f25cfbe829ee080807c6086d6edf8ec78c534/conda-ambertools-combine-pythons/build_ambertools.py#L44

**Comment by simonbray** [comment]
Hi @jchodera, there was already a request for that here: https://github.com/conda-forge/ambertools-feedstock/issues/4. I made a brief attempt to add them in this PR (https://github.com/conda-forge/ambertools-feedstock/pull/6) but haven't devoted any more time to it. Any PR is welcome.

**Comment by simonbray** [comment]
@hainm, would you be interested in taking over or joining me as maintainer of this feedstock? I guess you can help out with this kind of query far better than I can and there are probably a few things that could be improved in the recipe.

**Comment by hainm** [comment]
Hi Simon,

I am sorry I can’t not work on this. I don’t work for AMBER anymore so I
have limited availability for it. I currently only maintain the softwares I
wrote.

But do feel free to ask any questions.
Cheers.

**Comment by jchodera** [comment]
We should loop in @dacase and whoever the new AmberTools maintainers are.

Thanks for the input, @hainm!

**Comment by dacase** [comment]
I have some familiarity with building conda packages on OSX, for a subset of AmberTools.  I have zero experience with feedstocks.  Let me study the recipes and build scripts you have already created, and see how much I can understand.

**Comment by simonbray** [comment]
Yes; I tried it in this PR: https://github.com/conda-forge/ambertools-feedstock/pull/6. The Azure OSX build failed, and I don't have access to macOS locally to investigate further.

As far as I understand conda-forge requires compilers (and all other dependencies) to be included in this way. They are installed from the respective conda packages e.g. https://github.com/conda-forge/gfortran_osx-64-feedstock in this case.

**Comment by dacase** [comment]
I have OSX machines to try this on, but it may take a week or so to get to it, since my schedule is packed right now.  Feel free to ping me if you don't hear anything.

**Comment by dacase** [comment]
@simonbray @jchodera -- Sorry to be slow, and I may be posting this to the wrong issue(?)

Anyway, the issue here is this: when gfortran is installed from conda, our build script cannot compile and run a 3-line fortran program.  So the issue is not an Amber one, but probably that we don't know how to correctly set gfortran as a re-requisite for a conda build.

Simon asked for the enivironments on my OSX machine (10.15, Catailina) for when there is a gfortran prerequisite (which fails) and when I use my own compilers (which works, but doesn't appear to match a requirement that all compilers be installed from conda packages, as noted above).

Here is the environment when a use the conda packages:
```
AS=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-as
DIRTY=
AR=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-ar
LDFLAGS= -Wl,-rpath,/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib -L/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib
CODESIGN_ALLOCATE=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-codesign_allocate
PIP_NO_INDEX=True
CPU_COUNT=12
SSL_CERT_FILE=/Applications/ccp4-7.0/etc/ssl/cacert.pem
HOST=x86_64-apple-darwin13.4.0
NM=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-nm
CONDA_PERL=5.26
pin_run_as_build=OrderedDict([('python', {'max_pin': 'x.x', 'min_pin': 'x.x'}), ('r-base', {'max_pin': 'x.x', 'min_pin': 'x.x'})])
PYTHONNOUSERSITE=1
CPPFLAGS=-D_FORTIFY_SOURCE=2 -mmacosx-version-min=10.9 -isystem /Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/include
PAGESTUFF=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-pagestuff
CONDA_SHLVL=3
c_compiler=clang
LIPO=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-lipo
CONDA_PROMPT_MODIFIER=(/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env)
SIZE=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-size
NMEDIT=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-nmedit
PIP_CACHE_DIR=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/pip_cache
ignore_build_only_deps=set(['python', 'numpy'])
INSTALL_NAME_TOOL=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-install_name_tool
SEGEDIT=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-segedit
GFORTRAN=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-gfortran
SEG_HACK=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-seg_hack
PERL_VER=5.26
CLANGXX=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-clang++
LIBTOOL=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-libtool
CONDA_BACKUP_HOST=x86_64-apple-darwin13.4.0
CMAKE_GENERATOR=Unix Makefiles
PY_VER=2.7
CONDA_EXE=/Users/case/miniconda/bin/conda
cran_mirror=https://cran.r-project.org
REQUESTS_CA_BUNDLE=
OTOOL=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-otool
CONDA_STACKED_3=true
NPY_VER=1.11
CONDA_BUILD_STATE=BUILD
CONDA_R=3.5
DEBUG_FORTRANFLAGS=-march=nocona -mtune=core2 -ftree-vectorize -fPIC -fstack-protector -O2 -pipe -isystem /Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/include -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/work=/usr/local/src/conda/amber_phenix-0.9.6 -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold=/usr/local/src/conda-prefix -march=nocona -mtune=core2 -ftree-vectorize -fPIC -fstack-protector -O2 -pipe -Og -g -Wall -Wextra -fcheck=all -fbacktrace -fimplicit-none -fvar-tracking-assignments
PIP_NO_BUILD_ISOLATION=False
CXXFLAGS=-march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe -stdlib=libc++ -fvisibility-inlines-hidden -std=c++14 -fmessage-length=0 -isystem /Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/include -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/work=/usr/local/src/conda/amber_phenix-0.9.6 -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold=/usr/local/src/conda-prefix
PKG_BUILDNUM=0
CONDA_BUILD_SYSROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk
CONDA_BACKUP_LDFLAGS= -Wl,-rpath,/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib -L/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib
STRINGS=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-strings
_CE_CONDA=
CHECKSYMS=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-checksyms
CONDA_PREFIX_1=/Users/case/miniconda
CONDA_BUILD=1
PATH=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin:/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/bin:/Users/case/miniconda/condabin:/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env:/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin:/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold:/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/bin:/Users/case/miniconda/bin:/Users/case/phenix/build/bin:/opt/intel/compilers_and_libraries_2019.3.199/mac/bin/intel64:/Users/case/amber18/bin:/Applications/ccp4-7.0/etc:/Applications/ccp4-7.0/bin:.:/Users/case/bin:/Users/case/afnmr/bin:/usr/local/texlive/2018/bin/x86_64-darwin:/Users/case/misc_code/rdb:/opt/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin
SRC_DIR=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/work
fortran_compiler=gfortran
CONDA_PREFIX_2=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold
DEBUG_CXXFLAGS=-march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe -stdlib=libc++ -fvisibility-inlines-hidden -std=c++14 -fmessage-length=0 -Og -g -Wall -Wextra -isystem /Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/include -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/work=/usr/local/src/conda/amber_phenix-0.9.6 -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold=/usr/local/src/conda-prefix
LD=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-ld
CONDA_PREFIX=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env
BUILD=x86_64-apple-darwin13.4.0
F90=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-gfortran
PWD=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/work
STRIP=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-strip
cpu_optimization_target=nocona
PY3K=0
target_platform=osx-64
PKG_VERSION=0.9.6
F95=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-gfortran
LUA_VER=5
STDLIB_DIR=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python2.7
LANG=en_US.UTF-8
PIP_NO_DEPENDENCIES=True
CONDA_PY=27
SHLIB_EXT=.dylib
DEBUG_FFLAGS=-march=nocona -mtune=core2 -ftree-vectorize -fPIC -fstack-protector -O2 -pipe -isystem /Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/include -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/work=/usr/local/src/conda/amber_phenix-0.9.6 -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold=/usr/local/src/conda-prefix -march=nocona -mtune=core2 -ftree-vectorize -fPIC -fstack-protector -O2 -pipe -Og -g -Wall -Wextra -fcheck=all -fbacktrace -fimplicit-none -fvar-tracking-assignments
extend_keys=set(['ignore_build_only_deps', 'extend_keys', 'ignore_version', 'pin_run_as_build'])
F77=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-gfortran
ROOT=/Users/case/miniconda
REDO_PREBINDING=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-redo_prebinding
HTTPS_PROXY=
CXX=x86_64-apple-darwin13.4.0-clang++
CLANG=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-clang
_CE_M=
BUILD_PREFIX=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env
SHLVL=1
HOME=/Users/case
OSX_ARCH=x86_64
PKG_BUILD_STRING=placeholder
FORTRANFLAGS=-march=nocona -mtune=core2 -ftree-vectorize -fPIC -fstack-protector -O2 -pipe -isystem /Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/include -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/work=/usr/local/src/conda/amber_phenix-0.9.6 -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold=/usr/local/src/conda-prefix
PIP_IGNORE_INSTALLED=True
CFLAGS=-march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe -isystem /Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/include -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/work=/usr/local/src/conda/amber_phenix-0.9.6 -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold=/usr/local/src/conda-prefix
_CONDA_PYTHON_SYSCONFIGDATA_NAME=_sysconfigdata_x86_64_apple_darwin13_4_0
FC=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-gfortran
cxx_compiler=clangxx
HTTP_PROXY=
SP_DIR=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python2.7/site-packages
SYS_PREFIX=/Users/case/miniconda
r_base=3.5
INDR=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-indr
CONDA_PYTHON_EXE=/Users/case/miniconda/bin/python
CONDA_NPY=111
SUBDIR=osx-64
MACOSX_DEPLOYMENT_TARGET=10.15
NPY_DISTUTILS_APPEND_FLAGS=1
PREFIX=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold
R_VER=3.5
RECIPE_DIR=/Users/case/test/amber_phenix-0.9.6/recipe_ap
CONDA_LUA=5
PKG_CONFIG_PATH=/opt/intel/compilers_and_libraries_2019.3.199/mac/mkl/bin/pkgconfig
CONDA_DEFAULT_ENV=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env
DEBUG_CFLAGS=-march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe -Og -g -Wall -Wextra -isystem /Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/include -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/work=/usr/local/src/conda/amber_phenix-0.9.6 -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold=/usr/local/src/conda-prefix
RANLIB=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-ranlib
ARCH=64
CMAKE_PREFIX_PATH=:/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold
CC=x86_64-apple-darwin13.4.0-clang
SEG_ADDR_TABLE=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_build_env/bin/x86_64-apple-darwin13.4.0-seg_addr_table
LDFLAGS_LD=-pie -headerpad_max_install_names -dead_strip_dylibs -rpath /Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib -L/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib
SYS_PYTHON=/Users/case/miniconda/bin/python
PKG_NAME=amber_phenix
PKG_HASH=1234567
PYTHON=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/bin/python
FFLAGS=-march=nocona -mtune=core2 -ftree-vectorize -fPIC -fstack-protector -O2 -pipe -isystem /Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/include -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/work=/usr/local/src/conda/amber_phenix-0.9.6 -fdebug-prefix-map=/Users/case/miniconda/conda-bld/amber_phenix_1573483378128/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold=/usr/local/src/conda-prefix
_=/usr/bin/printenv
```


Next, here is the environment when the compilers are not listed as prerequisites in the meta.yaml file:
```
DIRTY=
PIP_NO_INDEX=True
CPU_COUNT=12
SSL_CERT_FILE=/Applications/ccp4-7.0/etc/ssl/cacert.pem
CONDA_PERL=5.26
pin_run_as_build=OrderedDict([('python', {'max_pin': 'x.x', 'min_pin': 'x.x'}), ('r-base', {'max_pin': 'x.x', 'min_pin': 'x.x'})])
PYTHONNOUSERSITE=1
CONDA_SHLVL=3
c_compiler=clang
CONDA_PROMPT_MODIFIER=(/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/_build_env)
PIP_CACHE_DIR=/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/pip_cache
ignore_build_only_deps=set(['python', 'numpy'])
PERL_VER=5.26
CMAKE_GENERATOR=Unix Makefiles
PY_VER=2.7
CONDA_EXE=/Users/case/miniconda/bin/conda
cran_mirror=https://cran.r-project.org
REQUESTS_CA_BUNDLE=
CONDA_STACKED_3=true
NPY_VER=1.11
CONDA_BUILD_STATE=BUILD
CONDA_R=3.5
PIP_NO_BUILD_ISOLATION=False
PKG_BUILDNUM=0
_CE_CONDA=
CONDA_PREFIX_1=/Users/case/miniconda
CONDA_BUILD=1
PATH=/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/_build_env/bin:/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/bin:/Users/case/miniconda/condabin:/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/_build_env:/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/_build_env/bin:/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold:/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/bin:/Users/case/miniconda/bin:/Users/case/phenix/build/bin:/opt/intel/compilers_and_libraries_2019.3.199/mac/bin/intel64:/Users/case/amber18/bin:/Applications/ccp4-7.0/etc:/Applications/ccp4-7.0/bin:.:/Users/case/bin:/Users/case/afnmr/bin:/usr/local/texlive/2018/bin/x86_64-darwin:/Users/case/misc_code/rdb:/opt/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin
SRC_DIR=/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/work
fortran_compiler=gfortran
CONDA_PREFIX_2=/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold
CONDA_PREFIX=/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/_build_env
BUILD=x86_64-apple-darwin13.4.0
PWD=/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/work
cpu_optimization_target=nocona
PY3K=0
target_platform=osx-64
PKG_VERSION=0.9.6
LUA_VER=5
STDLIB_DIR=/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python2.7
LANG=en_US.UTF-8
PIP_NO_DEPENDENCIES=True
CONDA_PY=27
SHLIB_EXT=.dylib
extend_keys=set(['ignore_build_only_deps', 'extend_keys', 'ignore_version', 'pin_run_as_build'])
ROOT=/Users/case/miniconda
HTTPS_PROXY=
_CE_M=
BUILD_PREFIX=/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/_build_env
SHLVL=1
HOME=/Users/case
OSX_ARCH=x86_64
PKG_BUILD_STRING=placeholder
PIP_IGNORE_INSTALLED=True
cxx_compiler=clangxx
HTTP_PROXY=
SP_DIR=/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python2.7/site-packages
SYS_PREFIX=/Users/case/miniconda
r_base=3.5
CONDA_PYTHON_EXE=/Users/case/miniconda/bin/python
CONDA_NPY=111
SUBDIR=osx-64
MACOSX_DEPLOYMENT_TARGET=10.15
NPY_DISTUTILS_APPEND_FLAGS=1
PREFIX=/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold
R_VER=3.5
RECIPE_DIR=/Users/case/test/amber_phenix-0.9.6/recipe_ap
CONDA_LUA=5
PKG_CONFIG_PATH=/opt/intel/compilers_and_libraries_2019.3.199/mac/mkl/bin/pkgconfig
CONDA_DEFAULT_ENV=/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/_build_env
ARCH=64
SYS_PYTHON=/Users/case/miniconda/bin/python
PKG_NAME=amber_phenix
PKG_HASH=1234567
PYTHON=/Users/case/miniconda/conda-bld/amber_phenix_1573483733793/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/bin/python
_=/usr/bin/printenv
```

Hope this helps someone....

**Comment by simonbray** [comment]
@dacase, thanks for reporting back. I'll try and find some time to look at this again over the next few weeks.

**Comment by jaimergp** [comment]
Solved by #18. Please note #20, though!
